scripts/change-log-generator.js

```diff
@@ -154,9 +169,15 @@ function buildMapFromCommit(commit) {
     var commitNum = COMMIT_REGEX.exec(commit);
     if (pr && commitNum) {
       var message = commit.replace(commitNum[0], '').replace(pr[0], '');
+      var type = TYPE_REGEX.exec(message);
       map[PR_NUM] = pr[0].replace(/[^\d]/g, '');
       map[COMMIT] = commitNum[0];
-      map[MESSAGE] = message.trim();
+      if (type) {
+        map[TYPE] = type[1];
+        message = message.replace(type[0], '');
+      }
+      message = message.trim();
+      map[MESSAGE] = message.charAt(0).toUpperCase() + message.slice(1);
```

Comment:
Since commitizen enforces that the first character is a lowercase for the commit message, capitalize it.